File: API/CineTECAPI/Models/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CineTECAPI.Models
{
    public class Cliente
    {
        public int cedula { get; set; }
        public string pnombre { get; set; }
        public string snombre { get; set; }
        public string apellido { get; set; }
        public int telefono { get; set; }
        public DateTime fechanac { get; set; }
        public int edad { get; set; }

    }
}

File: App movil/app/src/main/java/com/example/app/API/Cliente.kt
package com.example.app.API
// Clase encargada de administar los datos serializados de la cuenta en el API
data class Cliente(
    val cedula: Int,
    val pnombre: String,
    val snombre: String,
    val apellido: String,
    val telefono: Int,
    val fechanac: String,
    val edad: Int,
    val usuario: String,
    val contraseña: String
)